### ISSUE TITLE
Add Open Graph Support

### ISSUE THREAD

**Issue by krpeacock** [lanebreach/web_app#22]
In order for our links to grab attention, we should support the [Open Graph](https://ogp.me) standard and share a preview image with some descriptive text.

Use [React Helmet](https://github.com/nfl/react-helmet) to inject the content into the <head> of the application

**Comment by krpeacock** [comment]
@omegabytes If anyone wants to design a site preview image for us at the right dimensions, that would be rad

**Comment by lukemccrae** [comment]
Hey @krpeacock do we have prerendering turned on for the web app? I ran into a similar problem as [this issue](https://github.com/nfl/react-helmet/issues/442). Facebook isn't pulling data from the React Helmet head.

**Comment by krpeacock** [comment]
Yeah, we're definitely prerendering the page. You can check what we're
delivering through the "view source" option after right clicking the page.

Kyle Peacock
